### PR TITLE
remove TARGET_SEGMENTED from cgobj.c

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: src/backend/cgobj.c [hunk 1]
```diff
@@ -3200,11 +3183,7 @@ L1:     ;
 void Obj::write_long(int seg,targ_size_t offset,unsigned long data,
         unsigned lcfd,unsigned idx1,unsigned idx2)
 {
-#if TARGET_SEGMENTED
     unsigned sz = tysize[TYfptr];
-#else
-    unsigned sz = I64 ? 10 : 6;
```
Comment (yebblies) [inline src/backend/cgobj.c]: This doesn't matter?

Comment (WalterBright) [inline src/backend/cgobj.c]: Nope. cgobj.c is never used to generate 64 bit code.

Comment (yebblies) [inline src/backend/cgobj.c]: Oh right, cgobj.

**Comment by yebblies** [comment]
Auto-merge toggled on

**Pull request merged**